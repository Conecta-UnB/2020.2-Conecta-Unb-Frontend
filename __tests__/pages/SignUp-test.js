import React from 'react';
import { render } from '@testing-library/react';
import SignUpPage from '../../src/pages/SignUp';

describe('testando loginforms', () => {
  test('deve ser igual a snapshot', () => {
    const { asFragment } = render(<SignUpPage />);
    expect(asFragment(<SignUpPage />)).toMatchSnapshot();
  });
});
