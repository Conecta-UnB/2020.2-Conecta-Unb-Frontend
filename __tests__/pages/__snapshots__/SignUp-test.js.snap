// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testando loginforms deve ser igual a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="underHeader"
    >
      <h1>
        Cadastro
      </h1>
      <div
        id="formSignUp"
      >
        <div
          class="containerSignUpForm"
        >
          <form
            action="#"
          >
            <input
              name="nome"
              placeholder="Nome completo"
              value=""
            />
            <input
              name="matricula"
              placeholder="MatrÃ­cula"
              value=""
            />
            <input
              name="email"
              placeholder="E-mail"
              type="email"
              value=""
            />
            <input
              name="senha"
              placeholder="Senha"
              type="password"
              value=""
            />
            <input
              name="telefone"
              placeholder="Telefone"
              value=""
            />
            <button
              type="submit"
            >
              Cadastre-se
            </button>
          </form>
        </div>
        <img
          alt="signUpImage"
          src="[object Object]"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
